maliput/src/api/mod.rs

```diff
@@ -60,6 +99,47 @@ impl<'a> RoadGeometry<'a> {
             distance: maliput_sys::api::ffi::RoadPositionResult_distance(&rpr),
         }
     }
+    /// Get the lane matching given `lane_id`.
+    pub fn get_lane(&self, lane_id: &String) -> Lane {
+        unsafe {
+            Lane {
+                lane: maliput_sys::api::ffi::RoadGeometry_GetLane(self.rg, lane_id)
+                    .as_ref()
+                    .expect(""),
+            }
+        }
+    }
+    /// Get all lanes of the `RoadGeometry`.
+    /// Returns a vector of `Lane`.
+    /// # Example
+    /// ```rust, no_run
+    /// use maliput::api::RoadNetwork;
+    /// use std::collections::HashMap;
+    ///
+    /// let maliput_malidrive_plugin_path = maliput_sdk::get_maliput_malidrive_plugin_path();
+    /// std::env::set_var("MALIPUT_PLUGIN_PATH", maliput_malidrive_plugin_path);
+    /// let package_location = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+    /// let xodr_path = format!("{}/data/xodr/TShapeRoad.xodr", package_location);
+    /// let road_network_properties = HashMap::from([("road_geometry_id", "my_rg_from_rust"), ("opendrive_file", xodr_path.as_str())]);
+    /// let road_network = RoadNetwork::new("maliput_malidrive", &road_network_properties);
+    /// let road_geometry = road_network.road_geometry();
+    /// let lanes = road_geometry.get_lanes();
+    /// for lane in lanes {
+    ///    println!("lane_id: {}", lane.id());
+    /// }
+    /// ```
+    pub fn get_lanes(&self) -> Vec<Lane> {
+        let lanes = maliput_sys::api::ffi::RoadGeometry_GetLanes(self.rg);
+        let mut lanes_vec = Vec::new();
+        for lane in lanes {
+            unsafe {
+                lanes_vec.push(Lane {
+                    lane: lane.lane.as_ref().expect(""),
+                });
+            }
+        }
+        lanes_vec
```

Comment:
I think you can do the following:

```rust
lanes.into_iter().map(| l | Lane { lane: l.lane.as_ref().expect("") }).collect::<Vec<Lane>>()
```

Reply:
Nice! Keep dropping more rust good practices :smile:

Done.